### DISCUSSION TITLE
Actualização diária de casos e óbitos

### DISCUSSION THREAD

**Discussion by PedroLerias** [dssg-pt/covid19pt-data#1215] [Dados Nacionais e por ARS]
A DGS voltou a publicar uma actualização diária do número de novos casos e de óbitos por COVID-19.
Pode ser consultado diariamente aqui: https://covid19.min-saude.pt/numero-de-novos-casos-e-obitos-por-dia
Daria para completar a tabela com estes valores diários?
Obrigado pelo cuidado.

**Comment by davipt** [comment]
Olá Pedro.

Os novos dados são actualizados retroativamente, tal como acontecia com as amostras e as vacinas, portanto se tentar juntar os dados novos com os antigos, que eram fixados no dia que eram publicados, as contas nunca mais batem certo.

Em vez disso, e como não há mais dados para alem de casos confirmados e óbitos, optei por criar um ficheiro novo `dados_diarios.csv` que é simplesmente a conversão dos XLS originais para CSV, o mesmo processo aplicado aos dados de RT.

Assim cada um pode decidir fazer um merge entre os dois datasets e assumir que os valores novos não irão corresponder às somas por idades ou por regiões, quando esses dados existiam. Nem os totais batem certo, pois os relatórios semanais só têm o acréscimo da semana, e essas somas não batem certo dentro do próprio relatório.

**Reply by PedroLerias** [reply]
Obrigado, nunca cheguei a agradecer na altura a resposta.
E obrigado pela continuação na manutenção dos dados.
Boas Festas!
